articles/connections/references/apple-native/guides/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Add Sign In with Apple to Your Native App
+description: Learn how to add native login functionality to your native app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Native App
+
+To set up and configure Sign In with Apple for your native app, you will do the following:
+
+1. [Complete the prerequisites](#before-you-begin): Have an Apple Developer account (paid), an Auth0 tenant set up with a custom domain, and a web application configured to use Auth0 for authentication at that domain.  
+2. [Register your app in your Apple Developer account](#register-your-app-in-your-apple-developer-account): Obtain the following Apple IDs and keys for the application connection settings in the Dashboard. 
+    - For web apps: An Apple `ServiceID`
+    - For native apps: An Apple `AppID` ("Bundle Identifier")
+3. [Configure and test the application connection in Auth0](#configure-the-connection-in-auth0): Complete the social connection settings configuration information with the IDs and keys you obtained for your application from Apple. 
+
+## Prerequisites
+
+<%= include('../_includes/_siwa-prerequisites') %>
+
+## Register your native app on your Apple Developer account
+
+Once your Apple Developer account is set up, follow the instructions in the [Apple Setup Guide](/connections/apple-setup) to set up your application. 
```

Comment:
Note: Apple Setup Guide is web-app specific, yet referenced here in Native app setup guide.  Could be disorienting, as some of the steps do not match.